### PR TITLE
Replace __unicode__ method on all ReMo apps.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: remo/profiles/models.py [hunk 1]
```diff
@@ -237,10 +240,11 @@ class UserAvatar(caching.base.CachingMixin, models.Model):
 
     objects = caching.base.CachingManager()
 
-    def __unicode__(self):
+    def __str__(self):
         return "UserAvatar:%s" % self.user.userprofile.display_name
```
Comment (johngian) [inline remo/profiles/models.py]: Please fix double quotes.

Comment (johngian) [inline remo/profiles/models.py]: Please add ``u'`` prefix

**Comment by johngian** [comment]
r+wc

**Pull request merged**